src/modules/launcher/Plugins/Wox.Plugin.Folder/Main.cs

```diff
@@ -254,6 +255,7 @@ private static Result CreateOpenCurrentFolderResult(string incompleteName, strin
             var firstResult = "Open " + search;
 
             var folderName = search.TrimEnd('\\').Split(new[] { Path.DirectorySeparatorChar }, StringSplitOptions.None).Last();
+            var sanitizedPath = Regex.Replace(search, @"[\/\\]+", "\\");

```

Comment:
I think "C:/" is not covered in this regex

Reply:
I can think of other scenarios that are not covered as well, for example "C:\\" but Run opens root using "C:\\" and C:/"